lib/ruby_lsp/requests/workspace_symbol.rb

```diff
@@ -53,6 +43,23 @@ def perform
           )
         end
       end
+
+      private
+
+      def fuzzy_search
```

Comment:
Why this indirection ?

Reply:
What indirection are you referring to? I needed to pass a block to `Index#fuzzy_search`, and instead of

```rb
entries = @index.fuzzy_search(@query) do
  # ...
end
entries.flat_map do |entry|
  # ...
end
```

I chose to extract the first method invocation together with its block into a separate method.